native/arguments_test.go

```diff
@@ -144,8 +144,8 @@ func testArguments(t *testing.T, context spec.G, it spec.S) {
 			}.Configure(inputArgs)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(startClass).To(Equal(""))
-			Expect(args).To(HaveLen(5))
-			Expect(args).To(Equal([]string{"one", "two", "three", "more", "stuff"}))
+			Expect(args).To(HaveLen(4))
+			Expect(args).To(Equal([]string{"one", "two", "three", fmt.Sprintf("@%s",filepath.Join(ctx.Application.Path,"target/more-stuff.txt"))}))
 		})
 
 		it("works with quotes", func() {
```

Comment:
I'm not sure you need this test any more. The purpose was to verify that if something in the file was quoted, that when we merged the file into the list of arguments we still quote it properly. This isn't happening anymore, because we're just referencing the arguments file so this test ends up being basically the same as the previous test. I think you can just delete this test.

Reply:
Agreed!